src/app/checkout/step-2/credit-card/credit-card.spec.js

```diff
@@ -1,25 +1,33 @@
 import angular from 'angular';
 import 'angular-mocks';
-import omit from 'lodash/omit';
 import size from 'lodash/size';
+import find from 'lodash/find';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/observable/of';
+
 import module from './credit-card.component';
 
-import cartResponse from 'common/services/api/fixtures/cortex-cart-paymentmethodinfo-forms.fixture.js';
+import countriesResponse from 'common/services/api/fixtures/cortex-countries.fixture.js';
+import donorDetailsResponse from 'common/services/api/fixtures/cortex-donordetails.fixture.js';
 
 describe('checkout', () => {
   describe('step 2', () => {
     describe('credit card', () => {
       beforeEach(angular.mock.module(module.name));
       var self = {};
 
-      beforeEach(inject(($rootScope, $componentController, $httpBackend, $compile) => {
+      beforeEach(inject(($rootScope, $httpBackend, $compile) => {
         self.outerScope = $rootScope.$new();
         self.$httpBackend = $httpBackend;
 
         self.outerScope.submitted = false;
         self.outerScope.onSave = () => {};
         let compiledElement = $compile(angular.element('<checkout-credit-card submitted="submitted" on-save="onSave(success)"></checkout-credit-card>'))(self.outerScope);
+        // TODO: figure out how to skip calling $onInit when using $compile. For now, just expect the requests generated by calling $onInit. Adding a spy to $onInit would be best but I couldn't figure out how to access the controller before $onInit is called
```

Comment:
I can't seem to get a hold of it before it is run. It is nonexistent before the scope apply and then has already been run after